dms/static/description/index.html

```diff
@@ -0,0 +1,176 @@
+<section class="oe_container">
```

Comment:
This HTML file should be removed and regenerate it from the README fragments. Banner and other images should go out as well: marketing and old screenshots.

Reply:
Done